app/experimenter/legacy-ui/core/css/experimenter.css

```diff
@@ -244,3 +244,105 @@ pre.json {
   color: red;
   font-weight: bold;
 }
+
+.confidence-interval {
+  width: 70%;
+}
+
+.horizontal-line {
+  border-bottom: 3px solid black;
+  width: 100%;
+  display: block;
+}
+
+.vertical-line {
+  border-left: 3px solid black;
+  height: 10px;
+  width: 50%;
+  position: relative;
+  margin-left: 50%;
+}
+
+.positive-block {
+  height: 12px;
+  width: 35%;
+  background-color: green;
+  margin-top: -8%;
+  margin-right: 7%;
+  float: right;
+}
+
+.positive-number-wrapper {
+  width: 50%;
+  display: flex;
+  margin-left: 50%;
+}
+
+.positive-lower {
+  color: green;
+  width: 50%;
+  float: right;
+  text-align: left;
+}
+
+.positive-upper {
+  color: green;
+  text-align: right;
+  float: right;
+  width: 50%;
+}
+
+.negative-block {
+  height: 12px;
+  width: 35%;
+  background-color: red;
+  margin-top: -8%;
+  margin-left: 7%;
+}
+
+.negative-number-wrapper {
+  width: 50%;
+  display: flex;
+}
+
+.negative-lower {
+  color: red;
+  width: 50%;
+  float: left;
+  text-align: left;
+}
+
+.negative-upper {
+  color: red;
+  text-align: right;
+  width: 50%;
+}
+
+.neutral-block {
+  height: 12px;
+  width: 35%;
+  background-color: grey;
+  margin-top: -8%;
+  margin-left: 33%;
+  position: relative;
+  z-index: 10;
+}
+
+.neutral-number-wrapper {
+  width: 50%;
+  display: flex;
+  margin-left: 25%;
+}
+
+.neutral-lower {
+  color: grey;
+  width: 50%;
+  float: left;
+  text-align: left;
+}
+
+.neutral-upper {
+  color: grey;
+  text-align: right;
+  width: 50%;
+}
```

Comment:
There are some pretty specific values in these styles. Questions that come to mind are: will this look OK at different screen sizes, how fragile is it (e.g. if I need to change or add something down the road is everything else going to come crumbling down), and in elements that have strings of varying length will long text break anything? It looks like there is also a mixture of flexbox and floating elements in here; is this intentional or can we use flexbox everywhere? And in areas where flexbox and percentage widths are being used, can we use Bootstrap's [grid system](https://getbootstrap.com/docs/4.0/layout/grid/)?

Reply:
So I updated the PR and tried to use only the grid system. It's not the most beautiful thing (but I guess it wasn't before either) But I think the grid system addresses a bunch of these issues right? Lemme know what you think of these changes, and thank you for this feedback!